<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDFPageContentEditorWidget</class>
 <widget class="QDockWidget" name="PDFPageContentEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>607</height>
   </rect>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Content editor</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="toolGroupBox">
      <property name="title">
       <string>Toolbox</string>
      </property>
      <layout class="QGridLayout" name="toolGroupBoxLayout"/>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="adjustBox">
      <property name="title">
       <string>Geometry Tools</string>
      </property>
      <layout class="QGridLayout" name="geometryToolsGroupBoxLayout">
       <item row="0" column="0">
        <widget class="QToolButton" name="alignVertTopButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="alignVertMiddleButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="alignVertBottomButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="setSameWidthButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QToolButton" name="setSameHeightButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QToolButton" name="setSameSizeButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="alignHorLeftButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="alignHorMiddleButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="alignHorRightButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QToolButton" name="centerHorizontallyButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QToolButton" name="centerVerticallyButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QToolButton" name="centerRectButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="layoutGroupBox">
      <property name="title">
       <string>Layout Tools</string>
      </property>
      <layout class="QHBoxLayout" name="layoutToolBoxLayout">
       <item>
        <widget class="QToolButton" name="layoutHorizontallyButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="layoutVerticallyButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="layoutFormButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="layoutGridButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="appearanceGroupBox">
      <property name="title">
       <string>Appearance</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="contentGroupBox">
      <property name="title">
       <string>Content</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListView" name="listView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
